src/providers/export/exportProvider.ts

```diff
@@ -72,7 +73,18 @@ export abstract class ExportProvider<TOptions> implements IExportProvider {
     public async getAssetsForExport(): Promise<IAssetMetadata[]> {
         let predicate: (asset: IAsset) => boolean = null;
 
-        // @ts-ignore
+        const getProjectAssets = () => Promise.resolve(_.values(this.project.assets));
```

Comment:
This is the core of the fix.  Defaults to projectAssets when options is set to `Visited` or `Tagged`, otherwise merges in assets from the source connection when option is set to `All`.